driver/driver.go

```diff
@@ -50,14 +50,14 @@ const (
 )
 
 var (
-	command            = exec.Command
-	stat               = os.Stat
-	unmount            = syscall.Unmount
-	mount              = syscall.Mount
-	writeFile          = ioutil.WriteFile
-	mkdirAll           = os.MkdirAll
-	removeAll          = os.RemoveAll
-	hostname, anyerror = os.Hostname()
+	command   = exec.Command
+	stat      = os.Stat
+	unmount   = syscall.Unmount
+	mount     = syscall.Mount
+	writeFile = ioutil.WriteFile
+	mkdirAll  = os.MkdirAll
+	removeAll = os.RemoveAll
+	//hostname, anyerror = os.Hostname()
```

Comment:
same here. Can we remove if commented out lines are unused?

Reply:
kept it commented for reference if required